virttest/_wrappers.py

```diff
@@ -79,6 +79,8 @@ def lazy_import(name):
     :param name: Name of the module that is going to be imported
     :type name: String
     """
+    if name in sys.modules:
+        return sys.modules[name]
```

Comment:
Suggest splitting the fix into a seperated commit.

Reply:
Ok, makes sense.